doc/source/dev/contributor/building.rst

```diff
@@ -0,0 +1,182 @@
+:orphan:
+
+.. _building:
+
+Building from sources
+=====================
+
+.. note::
+
+   If you are only trying to install SciPy, see
+   `Installation <https://scipy.org/install>`__.
+
+Build instructions for different operating systems and an FAQ.
+
+.. _system-level:
+
+System-level dependencies
+-------------------------
+
```

Comment:
Let's add a note here that this is not needed when using `conda`, because it can install all required dependencies.

Reply:
The related notes under the OSX and Windows tabs can then be simplified or removed. Conda is recommended on Linux too.